### PR TITLE
Remove lock files from ITs

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
They are no longer up to date and we don't need them for integration tests.

### CODE REVIEW

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

File: analyzers/its/ReviewDiffs/packages.lock.json [hunk 1]
```diff
@@ -1,13 +0,0 @@
-{
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/ReviewDiffs/packages.lock.json:1]: @costin-zaharia-sonarsource This file gets regenerated when building the project and submitted to `git`. Shall it be restored?

Comment (costin-zaharia-sonarsource) [reply analyzers/its/ReviewDiffs/packages.lock.json:1]: No, I think we can ignore also this one. It's not part of the packages we ship.

Comment (pavel-mikula-sonarsource) [reply analyzers/its/ReviewDiffs/packages.lock.json:1]: Will you update .gitignore accordingly? Probably from
```
/analyzers/its/sources/**/packages.lock.json
```
to
```
/analyzers/its/**/packages.lock.json
```

Comment (costin-zaharia-sonarsource) [reply analyzers/its/ReviewDiffs/packages.lock.json:1]: Yes, will do so.

**Pull request merged**